NuGet.Config

```diff
@@ -16,6 +16,7 @@
     <add key="dotnet.myget.org dotnet-corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/" />    
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="dotnet.myget.org dotnet-cli" value="https://dotnet.myget.org/F/dotnet-cli/" />
+    <add key="dotnet.myget.org cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
```

Comment:
We shouldn't have other repos depending on this feed. Whatever packages this repo depends on, we should be explicit and take a dependency on their original feeds.

cc @eerhardt